tests/e2e/tests/plugins/PythonPlugin.spec.ts

```diff
@@ -36,19 +35,18 @@ const subRootFolder: string = '.vscode';
 
 const fileFolderPath: string = `${projectName}`;
 const tabTitle: string = 'hello-world.py';
-let workspaceName: string = 'python-hello-world';
 
 suite(`The 'PythonPlugin' test`, async () => {
     suite('Create workspace', async () => {
         test('Create workspace using factory', async () => {
             await browserTabsUtil.navigateTo(factoryUrl);
         });
 
-        projectAndFileTests.waitWorkspaceReadiness(projectName, subRootFolder);
+        workspaceHandlingTests.obtainWorkspaceNameFromStartingPage();
```

Comment:
It's strange to see code between the tests.

IMHO, it's part of `Wait until created workspace is started` test.